lib/instream/log/default_logger.ex

```diff
@@ -54,6 +56,14 @@ defmodule Instream.Log.DefaultLogger do
     entry
   end
 
+  def log(%DeleteEntry{} = entry) do
+    {conn, entry} = Map.pop!(entry, :conn)
+
+    _ = Logger.debug(["[delete] ", JSON.encode(entry.payload, conn), " predicate"])
```

Comment:
For consistency with the other log entries I would suggest to add the `:metadata` to the log entry and append them here:

```
    _ =
      Logger.debug(
        ["[delete] ", JSON.encode(entry.payload, entry.conn), " predicate"],
        metadata(entry)
      )
```

_I guess we can also skip the `Map.pop!/2` call as the reduced `entry` map is not used and we know the key exists in the passed struct._

Reply:
You are right. Thank you.